### ISSUE TITLE
Constructor parameter false case.

### ISSUE THREAD

**Issue by parloti** [jburzynski/TypeGen#99]
**The following code in C # ...**
```
public abstract class EnumerationDTO{
        protected EnumerationDTO(int id){
            Id = id;
        }
        public int Id { get; set; }
    }
```
```
public class ExecutionStatusDTO : EnumerationDTO{
        private ExecutionStatusDTO(int id) : base(id){}
        public static readonly ExecutionStatusDTO Open = new ExecutionStatusDTO(0);
}
```
**... generate the following Typescript ...**
```
export class EnumerationDTO {
    id: number;
    name: string;
}
```
```
export class ExecutionStatusDTO extends EnumerationDTO {
    static readonly open: ExecutionStatusDTO ={"Id":0};
}
```
**... which generates an error because the Id parameter is capitalized, while it should be lowercase.**

**Comment by jburzynski** [comment]
Hi, thanks for reporting this issue.

One way to work around this is to add [`TsDefaultValueAttribute`](https://typegen.readthedocs.io/en/latest/attributes.html#tsdefaultvalueattribute) (or use the `,DefaultValue()` method) for the member.

But of course this should work automatically, so it should be fixed.

**Comment by jburzynski** [comment]
This should already be fixed by #149 .
